pkg/cmd/cmd.go

```diff
@@ -209,3 +211,29 @@ func newFullCoverageCommand() *cobra.Command {
 
 	return cmd
 }
+
+func newTestCommand() *cobra.Command {
+	var (
+		repositoryPath string
+		compareBranch  string
+	)
+	cmd := &cobra.Command{
+		Use: "test",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			gocoverTest, err := gtest.NewGocoverTest(repositoryPath, compareBranch, cmd.OutOrStdout())
+			if err != nil {
+				return fmt.Errorf("new gocover test: %s", err)
+			}
+
+			err = gocoverTest.CheckGoTestFiles()
+			if err != nil {
+				return fmt.Errorf("check go test files: %s", err)
+			}
+			return nil
+		},
+	}
+
+	cmd.Flags().StringVar(&repositoryPath, "repository-path", "./", `the root directory of git repository`)
```

Comment:
this should be the package or module path？

Reply:
It's repository path. As diff will contains only contains relative path.